Tools/wasm/emscripten/browser_test/run_test.sh

```diff
@@ -0,0 +1,3 @@
+npm ci
```

Comment:
Does this file need a shebang?
```suggestion
#!/bin/sh
npm ci
```

Reply:
Also - is a chmod +x required?

Reply:
Not if we invoke it via `bash run_test.sh` =P but I will adjust it.